components/connectivity-adapter/internal/connectorservice/director/client.go

```diff
@@ -31,8 +28,6 @@ type client struct {
 }
 
 func (c client) GetApplication(systemAuthID string) (schema.ApplicationExt, apperrors.AppError) {
-
-	logrus.Info("System auth ID: " + systemAuthID)
```

Comment:
Please keep the log, it may be useful to debug some problems

Reply:
I think `systemAuthID` could be considered as sensitive data. I wanted to delete it then. You could always add it, rebuild an image and use it to debug your new implementation